tests/principals/wildcard.spec.ts

```diff
@@ -1,5 +1,5 @@
 import {expect} from 'chai';
-import {WildcardPrincipal} from '../../src/principals/wildcard';
+import {WildcardPrincipal} from '../../src';
```

Comment:
Should similar changes be made to other tests? I think it can be good for tests to import the external API where possible.

Reply:
Ah yes, indeed, this works too. We could indeed make similar changes in other tests.